src/Microsoft.ML.Transforms/CountFeatureSelectingTransformer.cs

```diff
@@ -2,28 +2,34 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Microsoft.ML.Core.Data;
 using Microsoft.ML.Runtime;
 using Microsoft.ML.Runtime.CommandLine;
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Runtime.EntryPoints;
 using Microsoft.ML.Runtime.Internal.Utilities;
-using Microsoft.ML.Transforms;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
+using Microsoft.ML.Transforms.FeatureSelection;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
 
-[assembly: LoadableClass(CountFeatureSelectingTransformer.Summary, typeof(IDataTransform), typeof(CountFeatureSelectingTransformer), typeof(CountFeatureSelectingTransformer.Arguments), typeof(SignatureDataTransform),
-    CountFeatureSelectingTransformer.UserName, "CountFeatureSelectionTransform", "CountFeatureSelection")]
+[assembly: LoadableClass(CountFeatureSelectingEstimator.Summary, typeof(IDataTransform), typeof(CountFeatureSelectingEstimator), typeof(CountFeatureSelectingEstimator.Arguments), typeof(SignatureDataTransform),
+    CountFeatureSelectingEstimator.UserName, "CountFeatureSelectionTransform", "CountFeatureSelection")]
 
-namespace Microsoft.ML.Transforms
+namespace Microsoft.ML.Transforms.FeatureSelection
 {
     /// <include file='doc.xml' path='doc/members/member[@name="CountFeatureSelection"]' />
-    public static class CountFeatureSelectingTransformer
+    public sealed class CountFeatureSelectingEstimator : IEstimator<ITransformer>
```

Comment:
>CountFeatureSelectingEstimator [](start = 24, length = 30)

I'm expecting @sfilipi to tell you to add sample here: [https://github.com/dotnet/machinelearning/tree/master/docs/samples/Microsoft.ML.Samples/Dynamic](https://github.com/dotnet/machinelearning/tree/master/docs/samples/Microsoft.ML.Samples/Dynamic)

Reply:
I am working on it now! 

---
In reply to: [235170327](https://github.com/dotnet/machinelearning/pull/1683#discussion_r235170327) [](ancestors = 235170327)

Reply:
Need to actually reference the samples now.

---
In reply to: [235225588](https://github.com/dotnet/machinelearning/pull/1683#discussion_r235225588) [](ancestors = 235225588,235170327)